### PR TITLE
Ford - Add OBDC Cable Length to CarPartList

### PR DESCRIPTION
**Pull request by sheaduncan**
- [x] added new part in selfdrive/car/ford/values.py and ran `selfdrive/car/docs.py` to generate new docs

### CODE REVIEW

**Comment by incognitojam** [comment]
It might make more sense to add the cable to the harness definition here: https://github.com/commaai/openpilot/blob/master/selfdrive/car/docs_definitions.py#L121

File: selfdrive/car/docs_definitions.py [hunk 1]
```diff
@@ -117,14 +117,16 @@ class CarHarness(EnumBase):
...
-  ford_q3 = BaseCarHarness("Ford Q3 connector", parts=[Accessory.harness_box, Accessory.comma_power_v2, Cable.rj45_cable_7ft, Cable.right_angle_obd_c_cable_1_5ft])
+  ford_q3 = BaseCarHarness("Ford Q3 connector", parts=[Accessory.harness_box, Accessory.comma_power_v2, Cable.rj45_cable_7ft])
   ford_q4 = BaseCarHarness("Ford Q4 connector", parts=[Accessory.harness_box, Accessory.comma_power_v2, Cable.rj45_cable_7ft, Cable.long_obdc_cable])
 
 
 class Device(EnumBase):
   threex = BasePart("comma 3X", parts=[Mount.mount, Cable.right_angle_obd_c_cable_1_5ft])
   # variant of comma 3X with angled mounts
-  threex_angled_mount = BasePart("comma 3X", parts=[Mount.angled_mount_8_degrees, Cable.right_angle_obd_c_cable_1_5ft])
+  threex_angled_mount = BasePart("comma 3X", parts=[Mount.angled_mount_8_degrees, Cable.right_angle_obd_c_cable_1_5ft]
```
Comment (incognitojam) [inline selfdrive/car/docs_definitions.py:127]:
```suggestion
  threex_angled_mount = BasePart("comma 3X", parts=[Mount.angled_mount_8_degrees, Cable.right_angle_obd_c_cable_1_5ft])
```

File: selfdrive/car/ford/values.py [hunk 2]
```diff
@@ -57,9 +57,14 @@ class FordCarInfo(CarInfo):
...
   def init_make(self, CP: car.CarParams):
+    # Harness in docs_definitions.py includes additional car_parts for Q3 and Q4
     harness = CarHarness.ford_q4 if CP.carFingerprint in CANFD_CAR else CarHarness.ford_q3
-    if CP.carFingerprint in (CAR.BRONCO_SPORT_MK1, CAR.MAVERICK_MK1, CAR.F_150_MK14):
+    if CP.carFingerprint in (CAR.BRONCO_SPORT_MK1, CAR.MAVERICK_MK1):
       self.car_parts = CarParts([Device.threex_angled_mount, harness])
+    elif CP.carFingerprint in (CAR.F_150_MK14, CAR.F_150_LIGHTNING_MK1):
+      self.car_parts = CarParts([Device.threex_angled_mount_nocable, harness])
+    elif CP.carFingerprint == CAR.MUSTANG_MACH_E_MK1:
+      self.car_parts = CarParts([Device.threex_angled_mount_nocable, harness, Cable.usbc_coupler])
```
Comment (incognitojam) [inline selfdrive/car/ford/values.py:67]: Does the Mach-E need an angled mount?

File: selfdrive/car/docs_definitions.py [hunk 3]
```diff
@@ -117,14 +117,16 @@ class CarHarness(EnumBase):
   nissan_a = BaseCarHarness("Nissan A connector", parts=[Accessory.harness_box, Cable.rj45_cable_7ft, Cable.long_obdc_cable, Cable.usbc_coupler])
   nissan_b = BaseCarHarness("Nissan B connector", parts=[Accessory.harness_box, Cable.rj45_cable_7ft, Cable.long_obdc_cable, Cable.usbc_coupler])
   mazda = BaseCarHarness("Mazda connector")
-  ford_q3 = BaseCarHarness("Ford Q3 connector")
-  ford_q4 = BaseCarHarness("Ford Q4 connector")
+  ford_q3 = BaseCarHarness("Ford Q3 connector", parts=[Accessory.harness_box, Accessory.comma_power_v2, Cable.rj45_cable_7ft])
```
[the remark this answers is not in the document]
Comment (sheaduncan) [reply selfdrive/car/docs_definitions.py:120]: Added for consistency, is this not correct?

File: selfdrive/car/ford/values.py (hunk 2 above)
Comment (sheaduncan) [reply selfdrive/car/ford/values.py:67]: This is my mistake, I misunderstood from a MachE user. Will update.

**Comment by sheaduncan** [comment]
Are all these github-actions because I sent it back to draft?

**Comment by adeebshihadeh** [comment]
> Are all these github-actions because I sent it back to draft?

Not your fault. It's a bug, and I just reverted it.

**Comment by sheaduncan** [comment]
Update:

I confirmed with the comma ops team that the 1.5ft right angle is shipped with every box. Therefore, I added the 9.5 ft cable for the F150, Lightning, and MachE, as well as the 1.5ft. Also included is the USBC coupler for use between 9.5ft cable and 1.5ft cable.

I updated the Q3 harness to be consistent with the Q4 harness. While it didn't need to be updated it makes sense to me to stay consistent in the approach.

File: selfdrive/car/docs_definitions.py [hunk 4]
```diff
@@ -117,8 +117,9 @@ class CarHarness(EnumBase):
...
```
Comment (sshane) [inline selfdrive/car/docs_definitions.py:120]:
```suggestion
  ford_q3 = BaseCarHarness("Ford Q3 connector")
```

File: selfdrive/car/ford/values.py [hunk 5]
```diff
@@ -57,8 +57,9 @@ class FordCarInfo(CarInfo):
   package: str = "Co-Pilot360 Assist+"
 
   def init_make(self, CP: car.CarParams):
+    # Harness in docs_definitions.py includes additional car_parts for Q3 and Q4
```
Comment (sshane) [inline selfdrive/car/ford/values.py:60]:
```suggestion
```

**Pull request merged**